tests/phpunit/tests/functions/wpuploadbits.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Test the wp_upload_bits function
+ *
+ * @group Functions
+ * @group Upload
+ * @covers ::wp_upload_bits
+ */
+
+class tests_wp_upload_bits extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57130
+	 *
+	 * @expectedDeprecated wp_upload_bits
+	 */
+	public function test_wp_upload_bits_should_throw_Deprecated_error_if_second_parm_present_and_not_null() {
+		wp_upload_bits( 'filename.txt', 'not_null', 'bits' );
+	}
+
+	/**
+	 * @ticket 57130
+	 *
+	 * return error if no filename pass in
+	 **/
+	public function test_wp_upload_bits_should_return_an_array_with_error_message_if_no_name_present() {
+		$this->assertSameSets( array( 'error' => __( 'Empty filename' ) ), wp_upload_bits( '', '', 'bits' ) );
+	}
+
+	/**
+	 * @ticket 57130
+	 *
+	 * return an array with error message in bad/no extension in the file name
```

Comment:
```suggestion
	 * Tests that wp_upload_bits() returns an array with an error message if the file type is not allowed.
	 *
	 * @ticket 57130
```
- Descriptions should appear at the top of the docblock.
- Docblock descriptions should use third-person singular verbs (i.e. "Tests", not "Test").